packages/cli/src/workflow-runner.ts

```diff
@@ -179,17 +179,14 @@ export class WorkflowRunner {
 			const postExecutePromise = this.activeExecutions.getPostExecutePromise(executionId);
 			postExecutePromise
 				.then(async (executionData) => {
-					if (this.externalHooks.exists('workflow.postExecute')) {
-						try {
-							await this.externalHooks.run('workflow.postExecute', [
-								executionData,
-								data.workflowData,
-								executionId,
-							]);
-						} catch (error) {
-							this.errorReporter.error(error);
-							this.logger.error('There was a problem running hook "workflow.postExecute"', error);
-						}
+					try {
+						await this.externalHooks.run('workflow.postExecute', [
```

Comment:
According to the check above we don't call this external hook for the recently introduced `evaluation` executions in regular mode. Should we?

Same applies to the recently moved `workflow-post-execute` event that was being emitted here.

Reply:
This is addressed in the follow up PR that consolidates all externalHook calls for executions.